### PR TITLE
docs(configuration options): document all config options for askui-ui…

### PR DESCRIPTION
**Pull request by thinh-askui**
…controller-server (#AS-979)

### CODE REVIEW

File: docs/versioned_docs/version-0.1.6/api/06-Configuration/askui-ui-controller-server.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+- [overWriteBinary](#overwritebinary)
+- [port](#port)
+
+## Properties
+
+### binaryVersion
+
+• **binaryVersion**: `string`
+
+There are different versions of The Askui-UI-Controller-Server.
```
Comment (mlikasam-askui) [inline docs/versioned_docs/version-0.1.6/api/06-Configuration/askui-ui-controller-server.md:33]: where can I find them?
  we need a table with different versions and the supported os and arch

File: docs/versioned_docs/version-0.1.6/api/06-Configuration/askui-ui-controller-server.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+
+The Host of the websocket server to connect via the runner-protocol. The default server is 127.0.0.1.
+___
+
+### logFilePath
+
+• **logFilePath**: `string`
+
+It is possible to specify a path for your log files. Log files are written to the console per default.
```
Comment (mlikasam-askui) [inline docs/versioned_docs/version-0.1.6/api/06-Configuration/askui-ui-controller-server.md:63]: we don't have access to the server console. when the controller is started with the lib

File: docs/docs/api/06-Configuration/askui-ui-controller-server.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
+---
+custom_edit_url: null
+---
+
+# Askui-UI-Controller-Server
```
[the remark this answers is not in the document]
Comment (adi-wan-askui) [reply docs/docs/api/06-Configuration/askui-ui-controller-server.md:5]: I agree with Samir. As we tackle the naming anyway this sprint, let's call it `askui UI Controller` like we discussed in our naming meeting. Make sure to lowercase `askui` (should always be lowercased, no matter where we use it).

File: docs/docs/api/06-Configuration/askui-ui-controller-server.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+
+Before running your tests you have start the Askui-UI-Controll-Server.
+There are multiple arguments which can be modified und provided to configure the Ask-UI-Contoller-Server. 
+In our implementation we use an interface which define all properties which can be provided to the Ask-UI-Contoller-Server.
+In the following we present you a short explanation for all properties.
+We also have an example to setup an controll server with specific values at the end of this page.
+
+## Table of contents
```
Comment (adi-wan-askui) [inline docs/docs/api/06-Configuration/askui-ui-controller-server.md:14]: Title Case for headings :)

**Review by adi-wan-askui — CHANGES_REQUESTED** [review]

File: docs/versioned_docs/version-0.1.6/api/06-Configuration/askui-ui-controller.md [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+If you start the askui Ui Controller you will see a small GUI where you can also select the monitor and identify the order
+of your monitors.
+___
+
+### host
+
+• **host**: `string` - Default: `127.0.0.1`
+
+
+The host the askui UI Controller is running on. This websocket server can be connected with the runner protocol.
```
Comment (adi-wan-askui) [inline docs/versioned_docs/version-0.1.6/api/06-Configuration/askui-ui-controller.md:104]: I would omit the second sentence. I don't think a user would get it as the runner protocol is our own protocol and the use of websocket communication is an implementation detail. The same goes for the port.

File: docs/versioned_docs/version-0.1.6/api/06-Configuration/askui-ui-controller.md [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+
+describe('jest with askui', () => {
+  
+  // Server for controlling the operating system
+  let askuiControlServer: AskuiControlServer;
+  
+  beforeAll(async () => {
+    askuiControlServer = new AskuiControlServer({
+    
+    // choosing the second montor
```
Comment (adi-wan-askui) [inline docs/versioned_docs/version-0.1.6/api/06-Configuration/askui-ui-controller.md:155]: _monitor_

File: docs/versioned_docs/version-0.1.6/api/06-Configuration/askui-ui-controller.md [hunk 7]
```diff
@@ -0,0 +1,170 @@
...
+  beforeAll(async () => {
+    askuiControlServer = new AskuiControlServer({
+    
+    // choosing the second montor 
+    display: 1,
+
+    // download the latest version of the server
+    binaryVersion: 'latest',
+
+    // start ask ui controller as minimized application
```
Comment (adi-wan-askui) [inline docs/versioned_docs/version-0.1.6/api/06-Configuration/askui-ui-controller.md:161]: _askui UI Controller_

**Review by adi-wan-askui — CHANGES_REQUESTED** [review]

File: docs/versioned_docs/version-0.1.6/api/06-Configuration/askui-ui-controller.md [hunk 8]
```diff
@@ -0,0 +1,170 @@
...
+    <td>&#10005;</td>
+    <td>&#10005;</td>
+    <td>&#10003;</td>
+  </tr>
+   <tr>
+    <td colspan="3" class="cap foot">
+        Availability of binary version for diffrent operating systems
+    </td>
+  </tr>
+</table>
```
[the remark this answers is not in the document]
Comment (adi-wan-askui) [reply docs/versioned_docs/version-0.1.6/api/06-Configuration/askui-ui-controller.md:78]: Added some info to our releasing docs: https://www.notion.so/askui/Releasing-442ddac471064772bc883f8e3e1dd689

File: docs/docs/api/07-configuration.md [hunk 9]
```diff
@@ -0,0 +1,22 @@
+---
```
[the remark this answers is not in the document]
Comment (thinh-askui) [reply docs/docs/api/07-configuration.md:1]: The main branch has the 07-configuration.md file. It's probably an old mistake by myself.  I will correct this in this pull request.

File: docs/versioned_docs/version-0.1.6/api/06-Configuration/askui-ui-controller-server.md [hunk 10]
```diff
@@ -0,0 +1,130 @@
...
+
+• **display**: `number`
+
+
+You can choose on which display you want to excecute all
+tests. You can only perform all tests on one display.
+The application which you want to test should be open and selected on your chosen display.
+
+The default value of display is `0` which is your main monitor. If you want to use your
+second monitor you can change the value to `1` (`2` for your third monitor etc.).
```
[the remark this answers is not in the document]
Comment (thinh-askui) [reply docs/versioned_docs/version-0.1.6/api/06-Configuration/askui-ui-controller-server.md]: Which chapter did you mean, maybe I overlook something?

[the remark this answers is not in the document]
Comment (mlikasam-askui) [reply docs/versioned_docs/version-0.1.6/api/06-Configuration/askui-ui-controller-server.md]: I am not sure which chapter, but I thought your first ticket was to document how to select the display. So I thought maybe you can link what you wrote back then.

**Comment by mlikasam-askui** [comment]
is the documentation of the client not included in this ticket?

**Review by adi-wan-askui — APPROVED** [review]
Good work :)

**Pull request merged**